whylog/config/__init__.py

```diff
@@ -109,12 +109,24 @@ def load_config(cls, path):
             return YamlConfig(**config_paths), assistant_class
 
     @classmethod
-    def create_file_with_config_paths(cls, config_paths, path_to_config):
+    def _create_file_with_config_paths(cls, config_paths, path_to_config):
         with open(path_to_config, 'w') as config_file:
             config_file.write(yaml.safe_dump(config_paths, explicit_start=True))
         return path_to_config
 
 
+class ConfigFactorySelector(object):
+    SUPPORTED_TYPES = {'yaml': YamlConfigFactory}
```

Comment:
Order of configs must be preserved. You will need to decide which one of many existing configs to load.